spark/src/main/java/org/apache/iceberg/actions/CommonActions.java

```diff
@@ -22,24 +22,16 @@
 import org.apache.iceberg.Table;
 import org.apache.spark.sql.SparkSession;
 
-public class Actions {
+public abstract class CommonActions {
```

Comment:
I think we use `Base` quite a bit in other places. Would it make sense to call it `BaseActions`?

Reply:
Sounds good to me